sdks/common/fn-api/src/test/resources/org/apache/beam/fn/v1/standard_coders.yaml

```diff
@@ -17,7 +17,7 @@
 
```

Comment:
*sigh* where were you guys on the original PR? Punish those who come after...

Done. PTAL.